src/multisig/sign.rs

```diff
@@ -21,4 +26,42 @@ impl MultisigService {
 
         Ok(signature)
     }
+
+    /// Given a `key_uid` and `pub_key`, find the matching mnemonic.
+    /// If `pub_key` is empty, use the currently active mnemonic.
+    pub(super) async fn find_matching_seed(
+        &self,
+        key_uid: &str,
+        pub_key: &[u8],
```

Comment:
Should we use `Option<&[u8]>` instead?  Conditioning on empty slice is less obvious to the caller.  Do we ever actually use this condition?